tests/snapshot.py

```diff
@@ -0,0 +1,9 @@
+import sys
+
+if sys.implementation.name == 'cpython':
+    from inline_snapshot import snapshot
+else:
```

Comment:
Any reason not to:

```suggestion
try:
    from inline_snapshot import snapshot
except ImportError:
```

? I mean, I've packaged `inline-snapshot` for Gentoo CPython already, but I suppose it may save others trouble for the time being.

Reply:
I got an idea. maybe we don't need this fix at all.
I can force inline-snapshot to run with `--inline-snapshot=disable` if something other then cpython is used. This would allow me to enable inline-snapshot again when I support it.

This would mean for you to package a inline-snapshot version for pypy which will not be able to create snapshots ... until I support it.

I think this would be better than to add a fix like this into every package which uses inline-snapshot.

Reply:
Sure — but could you make it possible to run a subset of inline-snapshot's test suite that covers the available functionality on PyPy then?

Reply:
Yes I will write some extra tests. But the disable logic is already tested.